scripts/validationAndPreparation.sh

```diff
@@ -47,7 +48,7 @@ export DATE_OF_NEWEST_CONTRIBUTION=$DATE_OF_NEWEST_CONTRIBUTION
 if [[ !  -z "${NEWEST_CONTRIBUTION}" ]]; then
 	cd /app/
 	echo "starting download; this could take a while..."
-	sftp -i /root/.ssh/id_rsa_worker validationworker@trusted-setup.staging.gnosisdev.com:$NEWEST_CONTRIBUTION /app/.
+	connet_server:$NEWEST_CONTRIBUTION /app/.
```

Comment:
This can be done a few more times below in this file

Reply:
will do :)